src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.cpp

```diff
@@ -21,6 +21,9 @@ ParamsKey ROIAlignKernelRef::GetSupportedKey() const {
     k.EnableDifferentTypes();
     k.EnablePoolType(PoolType::MAX);
     k.EnablePoolType(PoolType::AVG);
+    k.EnableRoiAlignedType(RoiAlignedType::ASYMMETRIC);
+    k.EnableRoiAlignedType(RoiAlignedType::HALF_PIXEL_FOR_NN);
+    k.EnableRoiAlignedType(RoiAlignedType::HALF_PIXEL);
```

Comment:
don't understand why you've added it here? Looks it's operation parameters specific, aren't they?

Reply:
They are) Done